pkgs/development/python-modules/fsspec/default.nix

```diff
@@ -17,7 +17,7 @@
 
 buildPythonPackage rec {
   pname = "fsspec";
-  version = "2022.5.0";
+  version = "2022.8.2";
```

Comment:
It's usually required to update [`gcsfs`](https://github.com/fsspec/gcsfs/blob/main/requirements.txt#L6) and others in the process.

Reply:
Hello !
Thanks for the feedback ! I've updated gcsfs.
I didn't run `nixpkgs-review rev HEAD` because there are way too many packages to rebuild for my poor computer (and big one like caffe). So instead I run 

```sh
nixpkgs-review rev HEAD  \
-p python310Packages.fsspec \
-p python3.10-gcsfs \
-p python3.10-ocifs \
-p python3.10-ossfs \
-p python3.10-s3fs \
-p python39Packages.fsspec \
-p python3.9-gcsfs \
-p python3.9-ocifs \
-p python3.9-ossfs \
-p python3.9-s3fs
```
And all tests pass. Do you think I am missing important packages ?